pkg/middlewares/accesslog/capture_response_writer.go

```diff
@@ -13,6 +13,20 @@ var (
 	_ middlewares.Stateful = &captureResponseWriter{}
 )
 
+type capturer interface {
+	http.ResponseWriter
+	Size() int64
```

Comment:
Is this implemented?

Reply:
Yes, line 66 (in old) or 90 (in new)?
Or are you talking about something else?

Reply:
Ah, I didn't see it that far down.

Reply:
happens even to the best of us ;)